### PR TITLE
Enh doc tidyup

### PR DESCRIPTION
**Pull request by to266**
- pycharm inspect fixes
 - updated copyright to 2015
 - changed all "Hyperspy" to "HyperSpy"

### CODE REVIEW

File: hyperspy/_signals/eds.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def _get_beam_energy(self):
         units_name = self.axes_manager.signal_axes[0].units
 
         if units_name == 'eV':
-            beam_energy = beam_energy * 1000
+            beam_energy *= 1000
         return beam_energy
```
Comment (vidartf) [inline hyperspy/_signals/eds.py]: Is it possible for `beam_energy` to be a reference (like an `ndarray`)? If so, this change might mean that you will change the value stored in the metadata: Previously the value of `beam_energy` was copied and reassigned, now it modifies the value "in place".

**Comment by vidartf** [comment]
I went through the diff, and except my comment about the compound operator (might be valid concern for other such replacements as well), it looks OK to me.

File: hyperspy/_signals/eds.py (hunk 1 above)
Comment (to266) [inline hyperspy/_signals/eds.py]: That's true. In this particular case it is not relevant, but I will go through and check

**Comment by to266** [comment]
@vidartf thanks for the review. Apparently the `beam_energy` is the most problematic - if it was indeed an array, I'd have to change it back. However it looks like it's always a number, so it's fine

**Comment by francisco-dlp** [comment]
The landscape report points at some genuine remaining problem.

**Comment by to266** [comment]
Attempted to fix the landscape problems, ready for review

**Pull request merged**